chainio/clients/wallet/fireblocks_wallet.go

```diff
@@ -159,6 +159,8 @@ func (t *fireblocksWallet) SendTransaction(ctx context.Context, tx *types.Transa
 		tx.Value().String(),       // amount
 		hexutil.Encode(tx.Data()), // calldata
 		replaceTxByHash,           // replaceTxByHash
+		// TODO: make this configurable
+		fireblocks.FeeLevelHigh, // feeLevel
```

Comment:
why not pass it as argument to SendTransaction?

Reply:
`SendTransaction` should share a common signature for all wallets. `feeLevel` field is specific to Fireblocks wallet.

Reply:
hmm I see. so when/how would you set the fee level then? Say if we calculate a gasPrice or whatever? Also what does adding this feeLevel do? Is fireblocks going to replace the gasPrice set in the tx??

Reply:
Right now, it sends every transaction with `HIGH` gas. There is a TODO for making it configurable. 
If you look at how `SendTransaction` works in this method, it only uses `To`, `Hash`, `Value`, and `Data` of the input transaction. So Fireblocks isn't aware of the gas specified in the `tx` object unless it's passed in in the API call. Right now, it's ignoring the gas fees set in the `tx` and always sending them with `HIGH` fee.

Reply:
gotcha, thanks for the explanation.